engine/apps/api/serializers/alert_group.py

```diff
@@ -189,17 +188,6 @@ def get_render_for_web(self, obj):
             AlertGroupWebRenderer,
         )
 
-    def get_render_for_classic_markdown(self, obj):
-        """Deprecated. TODO: remove"""
-        if not obj.last_alert:
-            return {}
-        return AlertGroupFieldsCacheSerializerMixin.get_or_set_web_template_field(
-            obj,
-            obj.last_alert,
-            AlertGroupFieldsCacheSerializerMixin.RENDER_FOR_CLASSIC_MARKDOWN_FIELD_NAME,
-            AlertGroupClassicMarkdownRenderer,
-        )
-
```

Comment:
this was marked as deprecated. It doesn't appear to be referenced on the frontend, removing it.